### PR TITLE
Load GenericFile from Solr for show view #146

### LINKED ISSUE #146: Generic File show page should use solr instead on Fedora
The GenericFile show page should use solr for showing the document instead of using the fedora document.
Here is where we used to load from solr before F4
https://github.com/psu-stewardship/scholarsphere/blob/develop/app/controllers/generic_files_controller.rb#L22-L29

Here is the removed code:
https://github.com/psu-stewardship/scholarsphere/blob/v2.1.1/app/controllers/generic_files_controller.rb#L12-L15

Here is some code in sufia that may help: https://github.com/projecthydra/sufia/issues/711

### PR DESCRIPTION
**Pull request by awead**
Fixes #146

### CODE REVIEW

File: app/views/generic_files/_show_details.html.erb [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+  <dd><%= @audit_status %></dd>
+  <% unless @generic_file.related_files.empty? %>
+  <dt>Related Files</dt>
+      <% @generic_file.related_files.each do |f| %>
+          <dd><%= link_to(f.label, sufia.generic_file_url(f)) %></dd>
+      <% end %>
+  <% end %>
+  <dt>Characterization</dt>
+  <dd>
+    <%= "not yet characterized" if characterization_terms.values.flatten.map(&:empty?).reduce(true) { |sum, value| sum && value } %>
```
Comment (carolyncole) [inline app/views/generic_files/_show_details.html.erb]: "characterization_terms.values.flatten.map(&:empty?).reduce(true) { |sum, value| sum && value }" would be a good candidate for a helper method

**Comment by carolyncole** [comment]
@awead  Looking good!  Do you think you would want to deploy this fix on Monday or Tuesday?

**Comment by carolyncole** [comment]
@awead I tried rerunning the travis failure.  It is in the GF show page.  Strange that it is only in the PR.

**Comment by awead** [comment]
@cam156 I updated it with a helper method and additional partial.

**Pull request merged**